default.toml

```diff
@@ -0,0 +1,18 @@
+max_width = 120
+ideal_width = 100
+leeway = 5
+tab_spaces = 4
+newline_style = "Unix"
+fn_brace_style = "SameLineWhere"
+fn_return_indent = "WithArgs"
+fn_args_paren_newline = true
+struct_trailing_comma = "Vertical"
+struct_lit_style = "BlockIndent"
+struct_lit_trailing_comma = "Vertical"
+enum_trailing_comma = true
+report_todo = "Always"
+report_fixme = "Never"
+reorder_imports = false
+expr_indent_style = "Tabbed"
+closure_indent_style = "Visual"
+single_line_if_else = false
```

Comment:
These are the same defaults as what rustfmt uses on itself.

On Wed, Aug 26, 2015, 22:31 Tobias Schottdorf <notifications@github.com>
wrote:

> In default.toml
> <https://github.com/Hoverbear/raft-rs/pull/108#discussion_r38063408>:
>
> > +leeway = 5
> > +tab_spaces = 4
> > +newline_style = "Unix"
> > +fn_brace_style = "SameLineWhere"
> > +fn_return_indent = "WithArgs"
> > +fn_args_paren_newline = true
> > +struct_trailing_comma = "Vertical"
> > +struct_lit_style = "BlockIndent"
> > +struct_lit_trailing_comma = "Vertical"
> > +enum_trailing_comma = true
> > +report_todo = "Always"
> > +report_fixme = "Never"
> > +reorder_imports = false
> > +expr_indent_style = "Tabbed"
> > +closure_indent_style = "Visual"
> > +single_line_if_else = false
>
> are the settings in this file taken from a "standard" defaults or based on
> anything?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Hoverbear/raft-rs/pull/108/files#r38063408>.
>
-- 

Sent from my mobile 5045076749
Alex Newman